Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs

```diff
@@ -1,47 +1,55 @@
-using UnityEngine;
 using UnityEditor;
 using UnityEditor.Callbacks;
+
+#if UNITY_IOS
+using UnityEngine;
 using System.IO;
+#endif // UNITY_IOS
 
-public class UnityARBuildPostprocessor
+namespace ARKit.Utils
```

Comment:
Is this something we are doing for future proofing? UnityARKit came from Unity

Reply:
There wasn't a namespace, and it seems these classes were added outside of the UnityARKit.

Reply:
I just matched it with the folder name.

Reply:
Looks like this came namespace-less from the plug-in:

https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/src/b4d4bbf5f4bfe454cad3fa2ecd897f5b785952ea/Assets/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs?at=default&fileviewer=file-view-default

Reply:
Grr. Well not sure what to do then.  I think it's bad practice to go namespaceless.

But to that end, did we take the package as is, or were there other edits?

Reply:
I agree about preferably not going namespaceless :/

As I understand, there were other edits to allow building for UWP. Likely, we should try to contribute those (and these namespace edits) back to the original repo.